### ISSUE TITLE
Feature request: Add a multiline text input field as custom fields types

### ISSUE THREAD

**Issue by WhiteWinterWolf** [pwndoc/pwndoc#283]
Currently there is no way to add a custom multiline text input field (same field type as the "Scope" or the "References" default fields).

The workaround is to use editors fields, but in this case their icons uselessly clutter the user interface, provides wrong expectations to the user, and makes parsing the resulting HTML data a bit more complex than raw multiline text.

**Comment by yeln4ts** [comment]
I don't think this type of field would be relevant, it needs to stay simple so if you have multiline text just use the editor so it can handle paragraph in World correctly. 
Scope and References are a specific case because they are arrays.

**Comment by WhiteWinterWolf** [comment]
*"Scope and References are a specific case because they are arrays."* -> precisely, and currently there is no clean way to handle other such "special cases" in the custom fields type.

For instance here I have a custom field to fill the list of accounts used during the audit. Currently I use a WYSIWYG field for this and then parse the result to turn HTML paragraph into array items and then process them. While this works, it doesn't seem very clean and it would be great to have the ability to have the same kind field as the Scope and Reference for such purpose.

**Comment by yeln4ts** [comment]
Oh ok I understand now what you meant, I thought you wanted to add a simple "textarea" field. 
Yes it would be nice to add this type of input field to the custom fields.
